projects/packages/forms/src/blocks/form-step-container/edit.js

```diff
@@ -0,0 +1,33 @@
+import { useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
+import AddStepControls from '../shared/components/form-add-step-controls';
+import StepControls from '../shared/components/form-step-controls';
+import useParentFormClientId from '../shared/hooks/use-parent-form-client-id';
+
+export default function StepContainerEdit( { clientId } ) {
+	const blockProps = useBlockProps( {
+		className: 'jetpack-form-step-container',
+	} );
+
+	const formClientId = useParentFormClientId( clientId );
+
+	// Ensure we have at least one step if empty
+	const innerBlocksProps = useInnerBlocksProps( blockProps, {
+		allowedBlocks: [ 'jetpack/form-step' ],
+		orientation: 'vertical',
```

Comment:
Do we need this? `orientation: 'vertical',`

Reply:
I don't think so, removed!